app/models/notification/center_popup.rb

```diff
@@ -0,0 +1,7 @@
+class Notification::CenterPopup < Notification
```

Comment:
Should be called an Activity Feed. I believe we put this design in the design slides before.